extensions/wikia/UserProfilePageV3/UserProfilePage.i18n.php

```diff
@@ -115,6 +115,7 @@
 	'user-identity-remove-fail' => 'There was some problem when removing the avatar!',
 	'user-identity-avatars-maintenance' => 'Upload and deletion of avatars temporarily disabled during maintenance!',
 	'userprofilepage-edit-modal-error' => 'Unexpected error occurred',
+	'user-identity-token-mismatch' => 'Oops! Token doesn\'t match',
```

Comment:
@kvas-damian I would really like to; I did not find anywhere for common messages, though.